lib/builderator/tasks/packer.rb

```diff
@@ -128,6 +128,42 @@ def wait(profile)
         say_status :complete, 'All copied images are available'
       end
 
+      desc 'remote_tag PROFILE', 'Apply existing tags to the AMI in remote AWS accounts'
+      def remote_tag(profile)
+        invoke :configure, [profile], options
+
+        sts_client = Aws::STS::Client.new(region: Config.aws.region)
+        allowed_cred_keys = %w(access_key_id secret_access_key session_token)
```

Comment:
@simonirwin-r7 maybe set this below just before using it? I couldn't tell you were using it until I `cmd+f` searched for it.